examples/nextjs/src/app/campaigns/[[...slug]]/page.js

```diff
@@ -0,0 +1,39 @@
+import { dotcmsClient, graphqlToPageEntity } from "@dotcms/client";
+import { MyPage } from "@/components/my-page";
+
+import { getGraphQLPageData } from "../../utils/gql";
+
+const client = dotcmsClient.init({
```

Comment:
maybe we can centralize this in other file later

Reply:
Thoughts @fmontes @zJaaal?

Reply:
I think I will need to centralize it for the content API Examples. Maybe is fine leave it here at the moment